pkg/controller/machinepool/machinepool_controller.go

```diff
@@ -698,7 +752,33 @@ func (r *ReconcileMachinePool) syncMachineSets(
 	for _, ms := range machineSetsToCreate {
 		logger.WithField("machineset", ms.Name).Info("creating machineset")
 		if err := remoteClusterAPIClient.Create(context.Background(), ms); err != nil {
-			logger.WithError(err).Error("unable to create machine set")
+			// Check error message
+			message := "Unable to create MachineSet"
+			if apierrors.IsAlreadyExists(err) {
+				rMS := machineapi.MachineSet{}
+				remoteClusterAPIClient.Get(context.Background(),
+					types.NamespacedName{
+						Name:      ms.Name,
+						Namespace: ms.Namespace,
+					},
+					&machineapi.MachineSet{})
+
+				// FIXME condense these into one?
+				if rMS.Labels[constants.HiveManagedLabel] != "true" {
+					logger.WithError(err).Errorf("%s. A MachineSet with the same name but without a hive managed label already exists."+
+						" To have hive manage this MachineSet, add the label \"hive.openshift.io/managed\" with the value \"true\" to the MachineSet.", message)
```

Comment:
Oh, no, let's not put adoption instructions into messages. We don't even know if we're going to officially support adoption, or to what level. For the moment, we want to keep it in our back pocket as a workaround for individual cases.